Podcast/SeriesGridCollectionViewCell.swift

```diff
@@ -61,7 +61,7 @@ class SeriesGridCollectionViewCell: UICollectionViewCell {
         titleLabel.text = series.title
         
         if showLastUpdatedText {
-            subscribersLabel.text = "Last updated " + Date.formatDateDifferenceByLargestComponent(fromDate: series.lastUpdated, toDate: Date())
+            subscribersLabel.text = "Last updated " + series.lastUpdatedAsString()
```

Comment:
Use the "Last updated \\(series.lastUpdatedAsString())" thing rather than plus.

Reply:
oh true haha yes much better